src/templates/DataSourcePage.jsx

```diff
@@ -129,7 +129,7 @@ export default function DataSourcePageTemplate({
               <ul className="highlight-ul m-t-50">
                 <li>
                   <b>Share insights from {frontmatter.name}</b> via link or
-                  Slack or embed them wherever you need to make your
+                 or embed them wherever you need to make your
```

Comment:
I think this results in two `or`s.

Reply:
Fixed 4667265891383a793f77b2295dd9eda02e99b4cb